app/Controllers/API/Donation/AddressChangeController.php

```diff
@@ -8,6 +8,7 @@
 use Symfony\Component\HttpFoundation\ParameterBag;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use WMDE\Fundraising\AddressChangeContext\Domain\Model\AddressType;
 use WMDE\Fundraising\AddressChangeContext\UseCases\ChangeAddress\ChangeAddressRequest;
 use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
 
```

Comment:
You can delete the `ADDRESS_TYPE_COMPANY` and `MESSAGE_NO_IDENTIFIER` consts from this class